server/api.go

```diff
@@ -958,7 +958,7 @@ func (a *API) enableNotifications(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	post.Message = "You'll now start receiving notifications here in Mattermost from chats and group chats from Microsoft Teams. To change this Mattermost setting, select **Settings > MS Teams**, or run the **/msteams notifications** slash command."
+	post.Message = "You'll now be notified of all MS Teams chats when you're away or offline from MS Teams. To turn off these notifications, go to Settings > MS Teams."
```

Comment:
```suggestion
	post.Message = "You'll now start receiving notifications in Mattermost for chats and group chats when you're away or offline in Microsoft Teams. To turn off these notifications in Mattermost, go to **Settings > MS Teams**. Learn more in our [documentation](https://mattermost.com/pl/ms-teams-plugin-end-user-learn-more)."
```

Reply:
@asaadmahmood can we post the GIF below the above message? ^ I think it's still valuable as a show and tell so the user understands what's expected now that they've connected their account